lib/build.js

```diff
@@ -129,7 +130,7 @@ module.exports = {
       this.customFileWatcher.close();
     }
     if (this.child) {
-      this.child.kill('SIGKILL');
+      kill(this.child.pid);
```

Comment:
I think this should still send 'SIGKILL' to the process, to forcefully terminate (this happens the second time the user aborts the build).

It appears to be easy with the `tree-kill` npm, just add 'SIGKILL' as second argument.